guides/kfp_tekton_install.md

```diff
@@ -55,16 +58,19 @@ To install the standalone Kubeflow Pipelines with Tekton, run the following step
 
 ## Kubeflow installation including Kubeflow Pipelines with Tekton Backend
 
-**Important: Please complete the [prequisites](#installation-targets-and-prequisites) before proceeding with the following instructions.**
+**Important: Please complete the [prerequisites](#installation-targets-and-prerequisites) before proceeding with the following instructions.**
 
-1. Follow the [Kubeflow install instructions](https://www.kubeflow.org/docs/ibm/deploy/install-kubeflow/#kubeflow-installation) to install whole Kubeflow stack with the kfp-tekton installation. Kubeflow v1.2.0 uses Tekton v0.14.0 and kfp-tekton v0.4.0.
+1. Follow the [Kubeflow install instructions](https://www.kubeflow.org/docs/ibm/deploy/install-kubeflow/#kubeflow-installation)
+   to install the entire Kubeflow stack with `kfp-tekton`.
+   Kubeflow `1.2.0` uses Tekton `v0.14.0` and `kfp-tekton` `v0.4.0`. <!-- TODO update-->
```

Comment:
@Tomcli -- are we working on an updated version of this installation process that uses the latest version of Kubeflow, Tekton, KFP-Tekton?

Reply:
Kubeflow (full stack) 1.3 will be releasing at end of Q1/early Q2. We will update it when it's time for the new official release.

Reply:
Okay, should I revert this line back to `Kubeflow v1.2.0 uses Tekton v0.14.0 and kfp-tekton v0.4.0`?

Reply:
sure, but I think you didn't change any wording other than the newlines. so it looks good to me.